### PR TITLE
Feature/error-fix-renderPointShare

### PR DESCRIPTION
**Pull request by Jakobimatagi**
# Pull Request Template

Go from a screen that selects the game play of single player or multiplayer.
When the single player mode is selected it goes into the screen which gives the option to read, write, and/or draw. 1. You will be able to check or uncheck draw or write but cannot choose both write and draw together 2. After chooseing you will be able to upload the drawing or writing to the platform.
When it is submited a function will pit the player against a randomly chosen boss.

## Description

Add a summary that address the following:

- motivation/purpose for the feature
  To make a gamemode menu for single player where they can choose read write or draw and then upload their writing or drawing
- What were those changes
  A gamemode menu for uploading a drawing by choosing the button.
- What issues were fixed
  A gamemode styled menu
- Provide the link to the user story it is referencing in Trello board or other supporting document
  https://trello.com/c/Q4Wusyn7/53-create-button-to-select-single-player-mode

## Commits Checklist

- [ ] commits have clear title names
- [ ] commit has descriptive message of what has changed
- [ ] more small commits than less big commits

## Code Checklist

- [ ] code is formatted appropriately
- [ ] code meets DRY standards
- [ ] no dead code (check for logs and print statements)
- [ ] no TODOS instead necessary comments
- [ ] code follows Story Squad standards for:
  - Language
  - Framework
  - Libraries

### PRs needs to be reviewed by at least 2 team members

- TPL is the 3rd reviewer
- Lots of comments and suggestions, please!

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

**Review by jimmoua1 — APPROVED** [review]
~~Good to go!~~

- Need to check conflicts

**Review by jayaramsivaramannair — APPROVED** [review]
Great catch, Jakobi!

File: pull_request_template.md [hunk 1]
```diff
@@ -1,38 +1,42 @@
 # Pull Request Template
```
Comment (tinomen) [inline pull_request_template.md:1]: This is a template file and not intended to be edited on every PR. Please revert the file back to this commit

https://github.com/Lambda-School-Labs/scribble-stadium-fe/commit/c4b22b16850ad5e0c7363f72804d326c80af1c74#diff-062b2db4de888830234713bb8f045b0d47b697dd1d6f64fced3e22f6e15d9642

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-    "compile-hero.disable-compile-files-on-did-save-code": true
-}
\ No newline at end of file
+  "compile-hero.disable-compile-files-on-did-save-code": true
```
Comment (tinomen) [inline .vscode/settings.json:2]: This affects everyone on the project using VSCode. Will you remove it or provide an explanation of the setting and why it's being changed in a special readme file (readme-vscode-config.md)

File: src/components/common/LoginCallbackLoader.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // istanbul ignore file
-import { LoginCallback } from '@okta/okta-react'; 
+import { LoginCallback } from '@okta/okta-react';
```
Comment (tinomen) [inline src/components/common/LoginCallbackLoader.js:2]: Looks like you still need to rebase the main branch into your branch. Okta was replaced with auth0 yesterday

**Review by tinomen — CHANGES_REQUESTED** [review]
There are a ton of linting formatting changes. Fixing lint errors can't be avoided but when there are a large number like this it would make sense to move the fixes to another branch and submit the lint fixes by themselves. There is so much noise that it's hard to see what is contributing to the actual renderPointShare fix.

**Pull request closed without merging**